.azure/obtemplates/build-linux.yml

```diff
@@ -16,39 +16,39 @@ jobs:
   - task: PowerShell@2
     displayName: Prepare Build Machine
     ${{ if eq(parameters.tls, 'openssl') }}:
-      target: linux_build_container2
+      target: linux_build_container2-x86_64
     ${{ else }}:
-      target: linux_build_container
+      target: linux_build_container-x86_64
```

Comment:
We can only use linux_build_container and linux_build_container2, which map to LinuxContainerImage and LinuxContainerImage respectively.

Reply:
where it is defined and how can we update it?

Reply:
We can't update or change it. It's part of 1ES template that we don't have control over.

Reply:
Perhaps you can override the value of LinuxContainerImage and LinuxContainerImage2 in the `variables` property of the job that the tasks belong to.

Reply:
but you still only have 2 linux container image you can use so if you wanna try that you need to make an extra job for the other build flavors.